frontend/apps/reader/modules/readermenu.lua

```diff
@@ -176,6 +176,23 @@ function ReaderMenu:setUpdateItemTable()
     end
 
     -- typeset tab
+    self.menu_items.reset_document_settings = {
+        text = _("Reset document settings to default"),
+        keep_menu_open = true,
+        callback = function()
+            UIManager:show(ConfirmBox:new{
+                text = _("Reset current document settings to default values?\n\nReading position, highlights and bookmarks will be kept.\nThe document will be reopened."),
```

Comment:
`to their default values` ? `to defaults` ? `to your default values` @Frenzie ?

We have `(The book will be reloaded.)` in readertoc.lua - sounds a bit better than `reopened` to my french ears.

Reply:
`to their default values` sounds good to me.

Reply:
> The book will be reloaded.

That was initially, but after this ConfirmBox we immediately have an infomessage "Opening file...", so I've changed to "reopened".